app/models/exercise/problem.rb

```diff
@@ -25,12 +25,23 @@ def expectations
     own_expectations + guide_expectations
   end
 
+  def custom_expectations
+    "#{own_custom_expectations}\n#{guide_custom_expectations}"
+  end
+
   def guide_expectations
     guide.expectations
   end
 
+  def guide_custom_expectations
+    guide.custom_expectations
+  end
+
   def evaluation_criteria?
-    manual_evaluation? || own_expectations.present? || test.present?
+    manual_evaluation? ||
+      own_expectations.present? ||
```

Comment:
Maybe we can extract `own_expectations.present? || own_custom_expectations.present?` to another method